docs/tracing.md

```diff
@@ -5,31 +5,29 @@ standardized [OpenTelemetry](https://opentelemetry.io/) traces for any of the su
 
 ## Example
 
-To enable tracing, call [`setup_tracing`][any_agent.tracing.setup_tracing] passing the selected framework.
+To enable tracing, add a TracingConfig object [`TracingConfig`][any_agent.config.TracingConfig] when creating an agent
 
 ```python
-from any_agent import AgentConfig, AgentFramework, AnyAgent
-from any_agent.tracing import setup_tracing
+from any_agent import AgentConfig, AgentFramework, AnyAgent, TracingConfig
 from any_agent.tools import search_web
 
 framework = AgentFramework("openai")
 
-setup_tracing(framework)
-
 agent = AnyAgent.create(
-        framework,
-        AgentConfig(
+        agent_framework=framework,
+        agent_config=AgentConfig(
                 model_id="gpt-4o",
-                tools=[search_web]
-            )
-        )
+                tools=[search_web],
+        ),
+        tracing=TracingConfig(enabled=True)
```

Comment:
um, this `enabled` doesn't exists (and I don't think it should)?

Reply:
Good catch... In order for our github action to catch errors in the md files, the code blocks have to be  ` ```python`, ` ```py` doesn't get checked 🤷